package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "helphub",
+  "description": "HelpHub Landing Page",
+  "version": "1.0.0",
+  "author": "Johan Singainy-Moutiapin <johan.singainy-moutiapin@opendatasoft.com>",
+  "private": false,
```

Comment:
It should be set as `private: true` to avoid some warning like `No license field` from NPM/Yarn

Reply:
@bost-h Updated !